src/platform/nm-linux-platform.c

```diff
@@ -5782,7 +5782,7 @@ link_set_sriov_params (NMPlatform *platform,
 	                                                                                  "device/sriov_drivers_autoprobe"),
 	                                                        10, 0, G_MAXUINT, 0);
 	if (   current_num == num_vfs
-	    && (autoprobe == -1 || current_autoprobe == autoprobe))
+	    && (autoprobe == NM_TERNARY_DEFAULT || current_autoprobe == autoprobe))
```

Comment:
I think, `nm_platform_sysctl_get_int_checked` above should only return 0, 1, and -1 (on error). Especially return `0` on error (above), is wrong.

Reply:
Fixed, thanks.